README.md

```diff
@@ -1,75 +1,49 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Capstone Project 1
 
-> One paragraph statement about the project.
+This is my first capstone project in Microverse, it is a Conference website!
 
-![screenshot](./app_screenshot.png)
+![screencapture-127-0-0-1-5500-index-html-2021-12-23-09_56_05](https://user-images.githubusercontent.com/87186552/147228339-414c3f56-0f6b-4edf-8190-a5633719c66e.png)
+![capstone-screenshot-mobile](https://user-images.githubusercontent.com/87186552/147228409-116a2acd-6130-48df-9008-44407be00612.png)
 
-Additional description about the project and its features.
+This project is for all screnn sizes, mobile and desktop views.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML
+- CSS
+- JAVASCRIPT
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://www.loom.com/share/3bc9d0dc05fc430ebbe3dd4d3e9f1352)
 
 
-## Getting Started
+👤 Author
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+GitHub: @Anny85-code
```

Comment:
- [ ] It would be nice to add more info about author, currently LinkedIn link is empty. This is what I include on my Authors section,  you can modify and use it 😉 

## Authors

```
👤 **Shoira**

- GitHub: [@shoirata](https://github.com/shoirata)
- LinkedIn: [LinkedIn](https://www.linkedin.com/in/shoira-tashpulatova-bab4a7122/)

![Shoira's GitHub stats](https://github-readme-stats.vercel.app/api?username=shoirata&count_private=true&theme=dark&show_icons=true)

```